Create FMI_Project_news.md (#122)
modelica/newsletter: _2019-03/FMI_Project_news.md

---
title: News from the FMI Project
author: MAP FMI

> category: "project"

Reviewer: ```suggestion
category: "association"
```

---

## FMI 2.0.1 released

After approval by the FMI Steering Committee and the Modelica Association members, version 2.0.1 of the FMI Specification 

> has been released today.

Reviewer: No relative dates. Please use absolute date here.

This is a maintenance release for FMI 2.0 with no new features, but bug fixes and clarifications to more than 50 issues.
FMUs created according to the FMI 2.0.1 specification are valid FMI 2.0 FMUs and vice versa.

The release notes and files to download can be found [here](https://github.com/modelica/fmi-standard/releases/tag/v2.0.1).

## Towards FMI 3.0

After acceptance by the FMI Steering committee, an “extra” directory for meta information has been added to the 
[FMI 3.0 features list in FMI FAQ](https://fmi-standard.org/faq/).
All FCPs (FMI Change proposals) described in the features list have now already been merged into the master branch of the 
[FMI standard repostory](https://github.com/modelica/fmi-standard). With this step, the API of the coming FMI 3.0 standard has been 
stabilized, so that prototyping of tools and improvement of the standard text can start now.

> The development of the FMI standard can be followed [on Github](https://github.com/modelica/fmi-standard) for the sources 

Reviewer: ```suggestion
The development of the FMI standard can be followed on [GitHub](https://github.com/modelica/fmi-standard) for the sources 
```

> The development of the FMI standard can be followed on [Github](https://github.com/modelica/fmi-standard) for the sources 

Reviewer: ```suggestion
The development of the FMI standard can be followed on [GitHub](https://github.com/modelica/fmi-standard) for the sources 
```

> of the standard text, headers and schema files, see also  the [generated documentation](https://fmi-standard.org/docs/3.0-dev/) for

Reviewer: ```suggestion
of the standard text, headers and schema files, see also the [generated documentation](https://fmi-standard.org/docs/3.0-dev/) for
```

the latest devlopment version.
You are welcome to sign the [CLA](https://svn.fmi-standard.org/fmi/branches/public/FMI_CCLA_v1.0_2016_06_21.pdf) of the FMI project 

> or the [CCLA](https://www.modelica.org/licenses/ModelicaAssociationCLA_1.1) fo the Modelica Assciation and comment and contribute 

Reviewer: ```suggestion
or the [CCLA](https://www.modelica.org/licenses/ModelicaAssociationCLA_1.1) of the Modelica Assciation and comment and contribute 
```

> or the [CCLA](https://www.modelica.org/licenses/ModelicaAssociationCLA_1.1) fo the Modelica Assciation and comment and contribute 

Reviewer: ```suggestion
or the [CLA](https://www.modelica.org/licenses/ModelicaAssociationCLA_1.1) of the Modelica Assciation and comment and contribute 
```
